react/components/CategoryItem.js

```diff
@@ -95,7 +95,7 @@ export default class CategoryItem extends Component {
   render() {
     return (
       <li
-        className={`${categoryMenu.itemContainer} flex items-center db list`}
+        className={`${categoryMenu.listItemContainer} flex items-center db list`}
```

Comment:
We cannot change a token name because some customizations are already depending on these tokens

Reply:
This css class `itemContainer` already exist in `ItemContainer.js`, that's why i renamed it, 
i hope purpose of using same css tokens both places is to reusability. 
Since this is separate HTML element i prefer using separate css token, so i will add new css token while we keeping the existing token as it was.